### DISCUSSION TITLE
asset imports from dynamic route

### DISCUSSION THREAD

**Discussion by m3tarayz** [vercel/next.js#13021] [Help] [answered]
Hi there, I am new to nextjs and attempting to migrate my cra project to it. Stumbled upon an issue with asset loading when using dynamic routes. Images and fonts, when loaded in a route component, lets say `home.tsx`, are working correctly and the url would be something like `http://localhost:3000/img/footer/facebook-logo.svg`.

However when using a route such as `search/[param].tsx`, the base url for all asset imports changes to `/search` and therefore does not point at the public folder. looks something like this - `http://localhost:3000/search/img/footer/facebook-logo.svg`.

all these assets are imported at the app level, eg in `_app.tsx`

Could someone please explain what I'm doing wrong in terms of setting up this route? 
Any help is much appreciated.
Thanks!

**Comment by rafinskipg** [comment, marked as answer]
@Samuel-Celko 

I imagine you have your assets under the *public* folder:
```
public/assets/logo.png
```

if that is the case you don't need to import them, you can do `img src="/assets/logo.png"` 

Just use the '/' as the first character. '/' stands for the root folder

**Reply by m3tarayz** [reply]
Thanks for the response, I was missing the `/` at the start indeed.
Many thanks!
